lib/signaling/v2/peerconnection.js

```diff
@@ -139,7 +152,9 @@ class PeerConnectionV2 extends StateMachine {
         value: null
       },
       _localMediaStream: {
-        value: new options.MediaStream()
+        value: isFirefox
```

Comment:
Actually, I think this needs to be gated on `isFirefox && RTCPeerConnection.prototype.addTransceiver`.